src/structures/GuildChannel.js

```diff
@@ -10,7 +10,7 @@ const Collection = require('../util/Collection');
 const { Error, TypeError } = require('../errors');
 
 /**
- * Represents a guild channel (i.g. a {@link TextChannel}, {@link VoiceChannel} or {@link CategoryChannel}).
+ * Represents a guild channel (i.g. a {@link TextChannel}, {@link VoiceChannel}, {@link CategoryChannel}, {@link NewsChannel} or {@link StoreChannel}).
```

Comment:
nit:
```suggestion
 * Represents a guild channel (i.g. a {@link TextChannel}, {@link VoiceChannel}, {@link CategoryChannel}, {@link NewsChannel}, or {@link StoreChannel}).
```

Reply:
Had only not included the comma as it wasn't there before. Also, both won't pass the ESLint check as it's over 120 characters on this line, suggestions (silly I didn't check beforehand), any suggestions?

Reply:
Updated my suggestion to break the line, however, I'm not sure if this will render as the same line on the website.

Reply:
I think it would be better to just have all of them on a new line and the line above detailing what channels they could be!

Reply:
I guess that works, I was also thinking of an unordered list, it'd take some space, but since it's the constructor's description, I think it'd be fine.

For instance:

```java
/**
 * Represents a guild channel from any of the following:
 * - {@link TextChannel}
 * - {@link VoiceChannel}
 * - {@link CategoryChannel}
 * - {@link NewsChannel}
 * - {@link StoreChannel}
 * @extends {Channel}
 */
```

Also, you missed to remove the `)` in ee3854b.

Reply:
That looks better, agreed